code/API_definitions/device-status.yaml

```
@@ -1,7 +1,73 @@
+      - For service delivery reasons, to ensure that the customer has access to particular service, and will not incur roaming charges in accessing them
+
+
+    # Relevant terms and definitions
+
+    * **Device**: A device refers to any physical entity that can connect to a network and participate in network communication.
+      At least one identifier for the device (user equipment) out of four options: IPv4 address, IPv6 address, Phone number, or Network Access Identifier assigned by the mobile network operator for the device.
+
+    * **Roaming** : Roaming status. `true`, if device is in roaming situation - `false` else.
+
+    * **Country** : Country code and name - visited country information, provided if the device is in roaming situation.
+
+
+    # API Functionality    
+
+    The API exposes following capabilities:
+
+    ## Device roaming situation
+
+    The endpoint `POST /roaming` allows to get roaming status and country information (if device in roaming situation) synchronously.
+
+    ## Device status subscription
+
+    These endpoints allow to manage event subscription on roaming device status event. 
+    The CAMARA subscription model is detailed in the CAMARA API design guideline document and follows Cloudevents specification.
+
+    It is mandatory in the subscription to provide the event `type` subscribed are several are managed in this API.
+
+    Following event ``type`` are managed for this API: 
+      - ``org.camaraproject.device-status.v0.roaming-status`` - Event triggered when the device switch from roaming ON to roaming OFF and conversely
+
+      - ``org.camaraproject.device-status.v0.roaming-on`` - Event triggered when the device switch from roaming OFF to roaming ON
+
+      - ``org.camaraproject.device-status.v0.roaming-off``: Event triggered when the device switch from roaming ON to roaming OFF
+
+      - ``org.camaraproject.device-status.v0.roaming-change-country``: Event triggered when the device in roaming change country code
+
+    Note: Additionnally to these list, ``org.camaraproject.device-status.v0.subscription-ends`` notification `type` is sent when the subscription ends. 
+    This notification does not require dedicated subscription. 
+    It is used when the subscription expire time (required by the requester) has been reached or if the API server has to stop sending notification prematurely. 
+
+    ### Notification callback
+
+    This endpoint describes the event notification received on subscription listener side when the event occurred. As for subscription, detailed description of the event notification is provided in the CAMARA API design guideline document.
+
+    **WARNING**: This callback endpoint must be exposed on the listener side as `POST /{$request.body#/webhook/notificationUrl}`
+      will be posted by the device status resource server to the webhook url provided by notification 
+      listener.
+
```

I think this section can be simplified:

```suggestion
    ### Notifications callback

    The  `notifications` callback describes the format of event notifications and expected responses to the messages sent when the event occurs. As for subscription, detailed description of the event notification is provided in the CAMARA API design guideline document.

    **WARNING**: This callback endpoint must be exposed and reachable on the listener side under 'notificationUrl` defined in the `webhook` attribute.

```